discovery_engine_core/web-api2/src/embedding.rs

```diff
@@ -42,3 +45,21 @@ impl Default for Config {
         }
     }
 }
+
+pub(crate) struct Embedder {
```

Comment:
:grey_exclamation:  Using a generic name allows to later one add alternative algorithms without needing to rename things everywhere. E.g. when using a different algorithm then bert for some customer using non-lating language.